Xamarin.Essentials/Browser/Browser.shared.cs

```diff
@@ -35,6 +35,60 @@ internal static Uri EscapeUri(Uri uri)
         }
     }
 
+    public readonly partial struct BrowserLaunchOptions : IEquatable<BrowserLaunchOptions>
+    {
+        public BrowserLaunchOptions(EssentialsColor? prefferedControlColor, EssentialsColor? preferredBackgroundColor, EssentialsColor? preferredTitleColor, bool? shouldShowTitle)
+        {
+            PrefferedControlColor = prefferedControlColor;
+            PreferredBackgroundColor = preferredBackgroundColor;
+            PreferredTitleColor = preferredTitleColor;
+            ShouldShowTitle = shouldShowTitle;
+        }
+
+        public EssentialsColor? PreferredTitleColor { get; }
+
+        public EssentialsColor? PreferredBackgroundColor { get; }
+
+        public EssentialsColor? PrefferedControlColor { get; }
+
+        public bool? ShouldShowTitle { get; }
+
+        public static bool operator ==(BrowserLaunchOptions lhs, BrowserLaunchOptions rhs) => lhs.Equals(rhs);
+
+        public static bool operator !=(BrowserLaunchOptions lhs, BrowserLaunchOptions rhs) => lhs.Equals(rhs);
+
+        public override bool Equals(object other) => other is BrowserLaunchOptions options && Equals(options);
+
+        public bool Equals(BrowserLaunchOptions other) => PreferredTitleColor.Equals(other.PreferredTitleColor)
+                                                          && PreferredBackgroundColor.Equals(other.PreferredBackgroundColor)
+                                                          && PrefferedControlColor.Equals(other.PrefferedControlColor)
+                                                          && ShouldShowTitle.Equals(other.ShouldShowTitle);
+
+        public override int GetHashCode() => (PreferredBackgroundColor, PreferredBackgroundColor, PrefferedControlColor, ShouldShowTitle).GetHashCode();
+    }
+
+    public readonly partial struct EssentialsColor : IEquatable<EssentialsColor>
```

Comment:
See this comment, but I think we need to discuss this further @redth @jamesmontemagno https://github.com/xamarin/Essentials/pull/646#issuecomment-446466659

Reply:
The TypeForwarding info you gave me was very interesting, I did not know about this feature before. Thank you for that! This indeed makes EssentialsColor obsolete. I will remove this in the next commit.